templates/CRM/Case/Page/DashBoard.tpl

```diff
@@ -49,65 +49,65 @@
-     0
-    {/if}
-    </td>
-   {/foreach}
-  </tr>
-  {/foreach}
-</table>
-{capture assign=findCasesURL}<a href="{crmURL p="civicrm/case/search" q="reset=1"}">{ts}Find Cases{/ts}</a>{/capture}
+    <h3>
+      {if $myCases}
+        {ts}Summary of Involvement{/ts}
+      {else}
+        {ts}Summary of All Cases{/ts}
+      {/if}
+    </h3>
+    <table class="report">
+      <tr class="columnheader">
+        <th>&nbsp;</th>
+        {foreach from=$casesSummary.headers item=header}
+          <th scope="col" class="right" style="padding-right: 10px;"><a href="{$header.url}">{$header.status}</a></th>
+        {/foreach}
+      </tr>
+      {foreach from=$casesSummary.rows item=row key=caseType}
+        <tr class="crm-case-caseStatus">
+          <th><strong>{$caseType}</strong></th>
+          {foreach from=$casesSummary.headers item=header}
+            {assign var="caseStatus" value=$header.status}
+            <td class="label">
+              {if $row.$caseStatus}
+                <a class="crm-case-summary-drilldown" href="{$row.$caseStatus.url}">{$row.$caseStatus.count}</a>
+              {else}
+                0
+              {/if}
+            </td>
+          {/foreach}
+        </tr>
+      {/foreach}
+    </table>
+    {capture assign=findCasesURL}<a href="{crmURL p="civicrm/case/search" q="reset=1"}">{ts}Find Cases{/ts}</a>{/capture}
 
-<div class="spacer"></div>
+    <div class="spacer"></div>
     <h3>{if $myCases}{ts}My Cases With Upcoming Activities{/ts}{else}{ts}All Cases With Upcoming Activities{/ts}{/if}</h3>
     {if $upcomingCases}
-    {include file="CRM/Case/Form/CaseFilter.tpl" context="$context" list="upcoming"}
-    <div class="form-item">
+      {include file="CRM/Case/Form/CaseFilter.tpl" context="$context" list="upcoming"}
+      <div class="form-item">
         {include file="CRM/Case/Page/DashboardSelector.tpl" context="dashboard" list="upcoming" all="$all"}
-    </div>
+      </div>
     {else}
-        <div class="messages status no-popup">
-      {ts 1=$findCasesURL}There are no open cases with activities scheduled in the next two weeks. Use %1 to expand your search.{/ts}
-        </div>
+      <div class="messages status no-popup">
+        {ts 1=$findCasesURL}There are no open cases with activities scheduled in the next two weeks. Use %1 to expand your search.{/ts}
+      </div>
     {/if}
 
-<div class="spacer"></div>
-    <h3>{if $myCases}{ts}My Cases With Recently Performed Activities{/ts}{else}{ts}All Cases With Recently Performed Activities{/ts}{/if}</h3>
-    {if $recentCases}
-    {include file="CRM/Case/Form/CaseFilter.tpl" context="$context" list="recent"}
-    <div class="form-item">
-        {include file="CRM/Case/Page/DashboardSelector.tpl" context="dashboard" list="recent" all="$all"}
-    </div>
-    {else}
+    <div class="spacer"></div>
+      <h3>{if $myCases}{ts}My Cases With Recently Performed Activities{/ts}{else}{ts}All Cases With Recently Performed Activities{/ts}{/if}</h3>
```

Comment:
So here - the <h3> is not inside the div & so it should be directly below (not indented)